doc/proxy-clients.rst

```diff
@@ -184,6 +184,42 @@ dispatcher as explained above and pass it to the Symfony client::
 
     $symfony = new Symfony($httpDispatcher);
 
+
+The `HttpDispatcher` sends real HTTP requests using any instance of `HttpAsyncClient`
+available in your application (see HTTPlug_ for more information). If you
+have a single node Symfony application setup you might want to directly route
+these requests through the kernel directly instead of having it execute real
+HTTP requests for e.g. performance reasons or just because it seems easier to
+setup as you don't have to know the domain/ip your application is reachable at.
+For that to work, you can pass an instance of `KernelClient` to your `$httpDispatcher`.
+It takes your kernel as the constructor argument which needs to implement
+`HttpCacheAwareKernelInterface` so the instance of `HttpCache` can be accessed
```

Comment:
accessed. (this is english, not german. lets make shorter sentences to make things easier to understand ;-) )